package.json

```diff
@@ -3,20 +3,22 @@
   "version": "1.1.0",
   "description": "",
   "scripts": {
-    "build": "rollup -c rollup.config.js --bundleConfigAsCjs"
+    "build:functionapp": "npm run functionapp:compile && npm run functionapp:zip",
+    "build:frontdoor": "npx perimeterx-azure-js-sdk -e src/enforcerConfig.json",
+    "functionapp:compile": "rollup -c rollup.config.js --bundleConfigAsCjs",
+    "functionapp:zip": "zip -r EnforcerFunction.zip . -x@.funcignore"
   },
   "dependencies": {
-    "perimeterx-azure-js-sdk": "latest"
+    "perimeterx-azure-js-sdk": "file:../perimeterx-azure-js-sdk"
```

Comment:
Include version with ^ (e.g., `^1.2.0`). We'll merge this after the enforcer is released.